### ISSUE TITLE
Breaks when trying to use u_camera

### ISSUE THREAD

**Issue by polyclick** [actarian/vscode-glsl-canvas#54]
I'm trying to use the camera uniform but the moment I include `u_camera` somewhere in the fragment shader, glsl-canvas breaks and stops updating altogether.

Here's a very simple example that show the problem:

```
#ifdef GL_ES
  precision highp float;
#endif

uniform vec3 u_camera;

void main() {
  vec3 c = vec3(u_camera.x, 1.0, 1.0);
  gl_FragColor = vec4(c, 1.0);
}
```

Cheers!

**Comment by jaredly** [comment]
I'm also experiencing this. Interestingly, even a comment that mentions `u_camera` breaks the view 🤔

**Comment by jaredly** [comment]
I think I tracked down the issue here https://github.com/actarian/glsl-canvas/pull/16

**Comment by jaredly** [comment]
here's a build of the extension that fixes it, fwiw
[glsl-canvas-0.2.12.vsix.zip](https://github.com/actarian/vscode-glsl-canvas/files/6479127/glsl-canvas-0.2.12.vsix.zip)

**Comment by actarian** [comment]
@polyclick @jaredly pull request has been published as version `0.2.13`
